src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -526,19 +526,10 @@ private SourceRepository GetSource(string source)
                 foreach (var package in packages)
                 {
                     // Check if the package is already installed
-                    if (package.Version == null)
+                    if (package.Version != null &&
```

Comment:
This is a behavior (potentially breaking) change in how the INstallPackage IVS API works, can you update the IVS method documentation.

Reply:
[Documentation](https://docs.microsoft.com/en-us/nuget/visual-studio-extensibility/nuget-api-in-visual-studio) always mentioned that it will install the latest version of the package if `version` is not provided, but we had a bug in our implementation.